package.json

```diff
@@ -108,12 +108,12 @@
     "react-ga4": "^2.1.0",
     "rxjs": "^7.8.1",
     "semver": "^7.6.0",
-    "tslib": "^2.5.0",
+    "tslib": "^2.6.2",
     "uuid": "^9.0.0",
     "winston": "^3.11.0"
   },
   "engines": {
-    "node": "^18.18.2"
+    "node": "^20.11.1"
```

Comment:
This is referenced in pm2-all.sh to stop startup if the system's current node version doesn't match. It appears this check isn't running though... maybe this is something for a follow up? We should validate when running `yarn start` and it'd be nice to validate in this in the CI as well. I wouldn't block on this though.